From 6c9bcee8f8ce6d5108b590a4b62207b26c13405c Mon Sep 17 00:00:00 2001
From: hwg <hwg@rock-chips.com>
Date: Sat, 7 Mar 2015 16:11:29 +0800
Subject: [PATCH] solve bluetooth hid problem:  when bluetooth hid device
 connected, the key is delayed to be reponse about 5 second

---
 arch/arm/configs/rockchip_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 1db7429ec0d5..c6d5d9b1c13d 100755
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -449,7 +449,6 @@ CONFIG_SND_RK_SOC_RT5631=y
 CONFIG_SND_RK_SOC_RT3224=y
 CONFIG_SND_RK_SOC_RK3036=y
 CONFIG_SND_RK_SOC_RK312X=y
-CONFIG_HID_BATTERY_STRENGTH=y
 CONFIG_HIDRAW=y
 CONFIG_UHID=y
 CONFIG_HID_A4TECH=y
-- 
2.35.3

